### PR TITLE
Ios styling

### PR DESCRIPTION
**Pull request by JasBuc**

### CODE REVIEW

File: native/KotlinIOS/KotlinIOS/DeparturesTableViewCell.swift [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+class DeparturesTableViewCell: UITableViewCell {
+
+    @IBOutlet weak var departureLabel: UILabel!
+    @IBOutlet weak var arrivalLabel: UILabel!
+    @IBOutlet weak var durationLabel: UILabel!
+    @IBOutlet weak var priceLabel: UILabel!
+    @IBOutlet weak var operatorLabel: UILabel!
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        self.accessibilityElements = [self.departureLabel, self.arrivalLabel, self.durationLabel, self.priceLabel, self.operatorLabel]
```
Comment (beccar97) [inline native/KotlinIOS/KotlinIOS/DeparturesTableViewCell.swift]: Using `.accessibilityElements` completely overrides the default accessibility behaviour of the cell, which can be a bit risky if we add a new element to the screen and forget to add it in here for example. Can you get the accessibility to work nicely by editing the accessibility traits of the elements from the Identity Inspector?

File: native/KotlinIOS/KotlinIOS/ViewController.swift [hunk 2]
```diff
@@ -21,19 +22,26 @@ class ViewController: UIViewController, ApplicationContractView {
         self.arrivalPicker.delegate = self
         self.arrivalPicker.dataSource = self
         setupTableView()
+        tableView.alpha = 0
     }
     
-    
     @IBAction func departureButton(_ sender: Any) {
```
Comment (beccar97) [inline native/KotlinIOS/KotlinIOS/ViewController.swift]: This is older code, but this isn't a brilliantly clear name for the action. In general, we try and name IBActions things like 'onHelpTapped', or 'onJourneySelected' which explain clearly the event this action is linked to

File: native/KotlinIOS/KotlinIOS/ViewController.swift [hunk 3]
```diff
@@ -47,6 +55,12 @@ extension ViewController: UIPickerViewDelegate, UIPickerViewDataSource {
     func pickerView(_ pickerView: UIPickerView, titleForRow row: Int, forComponent component: Int) -> String? {
         return stationData[row]
     }
+    func pickerView(_ pickerView: UIPickerView, didSelectRow row: Int, inComponent component: Int) {
+        let departureStation: String = stationData[departurePicker.selectedRow(inComponent: 0)]
+        let arrivalStation: String = stationData[arrivalPicker.selectedRow(inComponent: 0)]
+        presenter.setDepartureStation(departureStation: departureStation)
+        presenter.setArrivalStation(arrivalStation: arrivalStation)
+    }
```
Comment (Peter4137) [inline native/KotlinIOS/KotlinIOS/ViewController.swift]: I don't know if there's a way to split to two picker views without redoing the whole class but feels strange to be setting both variables when only one picker changes. Probably too much effort though to split them entirely

File: native/KotlinIOS/KotlinIOS/ViewController.swift [hunk 4]
```diff
@@ -21,19 +22,26 @@ class ViewController: UIViewController, ApplicationContractView {
...
+        tableView.isHidden = true
+        activityIndicatorView.startAnimating()
+        presenter.onButtonTapped()
     }
     
     func setLabel(text: String) {
         label.text = text
     }
     
+    func createAlertMessage(alertMessage: String){
```
Comment (Peter4137) [inline native/KotlinIOS/KotlinIOS/ViewController.swift]: Space needed between ) and {

**Review by Peter4137 — CHANGES_REQUESTED** [review]
Looks really good to me

File: SharedCode/src/commonMain/kotlin/ApplicationPresenter.kt [hunk 5]
```diff
@@ -73,7 +73,7 @@ class ApplicationPresenter: ApplicationContract.Presenter() {
         val departureDateTime = processTimeForDisplay(journeyDetails.departureTime)
         val arrivalDateTime = processTimeForDisplay(journeyDetails.arrivalTime)
         val journeyTime: TimeSpan = arrivalDateTime - departureDateTime
-        val journeyTimeMinutes: String = "${journeyTime.minutes}m"
+        val journeyTimeMinutes: String = "${journeyTime.minutes.toInt()} min"
```
Comment (beccar97) [inline SharedCode/src/commonMain/kotlin/ApplicationPresenter.kt:75]: If you want to use  just m in the display, but want min in the voiceover you could look at using the `accessibilityLabel` trait 🙂

File: native/KotlinIOS/KotlinIOS/ViewController.swift [hunk 6]
```diff
@@ -21,21 +22,26 @@ class ViewController: UIViewController, ApplicationContractView {
         self.arrivalPicker.delegate = self
         self.arrivalPicker.dataSource = self
         setupTableView()
+        tableView.alpha = 0
```
Comment (beccar97) [inline native/KotlinIOS/KotlinIOS/ViewController.swift]: What's the intention behind this line? Are you trying to hide the table?

**Review by beccar97 — COMMENTED** [review]
Looks good! Just a few things :)

File: native/KotlinIOS/KotlinIOS/ViewController.swift (hunk 6 above)
Comment (JasBuc) [reply native/KotlinIOS/KotlinIOS/ViewController.swift]: I believe so, I've changed so now use `tableView.isHidden = true` like is used elsewhere

Comment (beccar97) [reply native/KotlinIOS/KotlinIOS/ViewController.swift]: Good plan, that's a bit clearer to anyone reading the code 🙂

File: native/KotlinIOS/KotlinIOS/DeparturesTableViewCell.swift (hunk 1 above)
Comment (JasBuc) [reply native/KotlinIOS/KotlinIOS/DeparturesTableViewCell.swift]: Here I'm trying to get VoiceOver to read the components in the right order - am I able to set the hierarchy of elements from the Identity Inspector to achieve this? Or is there a different way I should go about doing this?

Comment (beccar97) [reply native/KotlinIOS/KotlinIOS/DeparturesTableViewCell.swift]: Ah okay, annoyingly it seems like custom table cells are one of the times that using `accessibilityElements` is basically the only way of getting voice over to work properly. Sorry!

Comment (JasBuc) [reply native/KotlinIOS/KotlinIOS/DeparturesTableViewCell.swift]: Ah okay - annoying element! No worries! :)

**Pull request merged**